extmod/modssl_mbedtls.c

```diff
@@ -264,6 +395,40 @@ STATIC void ssl_context_load_key(mp_obj_ssl_context_t *self, mp_obj_t key_obj, m
     }
 }
 
+
+STATIC mp_obj_t ssl_context_load_certchain(size_t n_args, const mp_obj_t *pos_args,
```

Comment:
CPython's signature of this method is:
```py
SSLContext.load_cert_chain(certfile, keyfile=None, password=None)
```
But it looks like mbedtls requires both cert and key.  So MicroPython's signature should be:
```py
SSLContext.load_cert_chain(certfile, keyfile)
```
Ie both arguments are required.

So please update this code here to implement this signature:
```c
STATIC mp_obj_t ssl_context_load_cert_chain(mp_obj_t self_in, mp_obj_t certfile, mp_obj_t keyfile) {
    ...
}
STATIC MP_DEFINE_CONST_FUN_OBJ_3(ssl_context_load_cert_chain_obj, ssl_context_load_cert_chain);
```

Reply:
I think this breaks CPython compatibility? 

```C
STATIC mp_obj_t ssl_context_load_cert_chain(mp_obj_t self_in, mp_obj_t certfile, mp_obj_t keyfile) {
    ...
}
STATIC MP_DEFINE_CONST_FUN_OBJ_3(ssl_context_load_cert_chain_obj, ssl_context_load_cert_chain);
```
This doesn't allow keyword args, e.g. this in MicroPython will fail
```py
SSLContext.load_cert_chain("cert.der", keyfile="key.der")
```

I modified `keyfile` arg and made it `MP_ARG_REQUIRED`, this should allow both forms, right?

```py
SSLContext.load_cert_chain(certfile, keyfile=None)
```
or
```py
SSLContext.load_cert_chain(certfile, keyfile)
 ```